### ISSUE TITLE
Feature Request: 2x Scale everything

### ISSUE THREAD

**Issue by creamcast** [ClassicUO/ClassicUO#1254]
How hard would this be to implement? On a full HD monitor (1920x1080) everything is tiny. Scaling container and zooming helps but everything else remains small. Would be a great feature to have to be able to scale everything by 2x if its not too difficult.

**Comment by birdinforest** [comment]
Agree. I find it is hard to read message and dialogue on high resolution screen. Especially when the text is in red.
I would like to work out a feature to set font size. 

According to my understanding, it loads font from game files, then generate font textures on basis of the font size and text length. It will be appreciated if someone can make a brief intoduction about details.

**Comment by Arutosio** [comment]
I also have the same problem, it would be very useful to implement a function that allows you to use custom fonts and the function to choose the font size and that the game can adapt to it.
I found this repository with very good quality UO fonts.
I think this implementation has a high priority since we have already reached 4k
https://github.com/uo-clife-work/CLF-UoClassic-Font

**Comment by SaschaKP** [comment]
I would bet that a global scaler maybe is more useful, but making the fonts and whatever is possible more "scale" friendly is really the best way, maybe we can mimic the way that was used by mobileUO to scale the whole game? and also fix the scaling issues, since scaled objects are pixellated

**Comment by dirkson** [comment]
I find the current client unusable at its current scale on a 1440P monitor. +1 for feature.

**Comment by mandlar** [comment]
I can potentially upstream a PR of how MobileUO scales the entire game if that would be beneficial

/cc @andreakarasho
